### ISSUE TITLE
zero_module

### ISSUE THREAD

**Issue by MsQueeniee** [openai/guided-diffusion#152]
为什么要在每一个block中使用zero_module将网络参数清零

Why do we need to use zero_module to zero out the network parameters in each block?

def zero_module(module):
    """
    Zero out the parameters of a module and return it.
    """
    for p in module.parameters():
        p.detach().zero_()
    return module
